recipe/build.sh

```diff
@@ -14,6 +14,14 @@ FORCE_CUDA=0
 if [ $build_type == "cuda" ]
 then
     export FORCE_CUDA=1
+    # Create symlinks of cublas headers into CONDA_PREFIX
+    if [ ! -d "$CONDA_PREFIX/include" ] 
```

Comment:
You shouldn't need this check, since you are using the `-p` option for `mkdir`

Reply:
Yeah, I'd thought so and I didn't have it in the first place. But when it failed, I'd to add that check. I'll retry.

Reply:
Done.